tests/tests.json

```diff
@@ -701,6 +708,175 @@
       "uri-from-cri": "?a&b&c",
       "resolved-cri": "84218263666f6f1912678262706162746883616161626163",
       "resolved-uri": "coaps://foo:4711/pa/th?a&b&c"
+    },
+    {
+      "description": "Empty CRI",
+      "uri": null,
```

Comment:
This is incorrect: 'As a special case, an empty array is sent in place for a remaining [0] (URI "")' -- should check whether that's consistent all over the text / where this interpretation came from.

Reply:
Didn't find any location that indicated something else; the latest update fixes the case in alignment with the Rust implementation.